app/views/system/inputs/extractors/configurations/regex_replace.scala.html

```diff
@@ -0,0 +1,75 @@
+@(extractor: org.graylog2.restclient.models.Extractor = null)
+@regex = @{
+    if(extractor == null) {
+        ""
+    } else {
+        extractor.getExtractorConfig.get("regex")
+    }
+}
+@replacement = @{
+    if(extractor == null) {
+        "$1"
+    } else {
+        extractor.getExtractorConfig.get("replacement")
+    }
+}
+@replace_all = @{
+    if(extractor == null) {
+        false
+    } else {
+        Boolean.parseBoolean(String.valueOf(extractor.getExtractorConfig.get("replace_all")))
+    }
+}
+
+<input type="hidden" name="extractor_type" value="regex_replace" />
+
+<div class="form-group">
+    <label for="regex_value">Regular expression:</label>
+    <div class="row">
+        <div class="col-md-5">
+            <div class="input-group">
+                <input name="regex" id="regex" type="text"
+                placeholder="^.*string(.+)$" class="form-control" value="@regex" required />
+                <span class="input-group-btn">
+                    <button class="btn btn-info xtrc-try-regex-replace" type="button">Try!</button>
+                </span>
+            </div>
+        </div>
+    </div>
+    <span class="help-block">
+        The regular expression used for extraction.
+        Learn more in the @views.html.partials.links.docs(views.helpers.DocsHelper.PAGE_EXTRACTORS, "documentation").
+    </span>
+</div>
+
+<div class="form-group">
+    <label for="regex_value">Replacement:</label>
+    <div class="row">
+        <div class="col-md-5">
+            <div class="input-group">
+                <input name="replacement" id="replacement" type="text"
+                placeholder="$1" class="form-control" value="@replacement" required />
+            </div>
+        </div>
+    </div>
+    <span class="help-block">
+        The replacement used for the matching text.
```

Comment:
Could we also add some more information about the replacement value you can use? Probably we should write about it in the documentation, but at least we should point to the [API](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Matcher.html#replaceAll-java.lang.String-).